pyvista/common.py

```diff
@@ -199,7 +210,7 @@ def t_coords(self, t_coords):
             raise AssertionError('Number of texture coordinates ({}) must match number of points ({})'.format(t_coords.shape[0], self.n_points))
         if t_coords.shape[1] != 2:
             raise AssertionError('Texture coordinates must only have 2 components, not ({})'.format(t_coords.shape[1]))
-        if np.min(t_coords) < 0.0 or np.max(t_coords) > 1.0:
+        if np.min(t_coords) < 0.0 or np.max(t_coords) > 1.0 and not self._repeat_texture:
```

Comment:
Hm... if we don't have the `repeat_texture` property on the mesh itself then this will be problematic.

Perhaps we could change this to a warning:

```py
warnings.warn(...)
```